src/ninchatclient/lib/call.go

```diff
@@ -6,6 +6,10 @@ import (
 )
 
 func call(params map[string]interface{}, onLog *js.Object, address string) *js.Object {
```

Comment:
The correct way to detect undefined JavaScript value is to change the Go function signature to
```
func call(params map[string]interface{}, onLog *js.Object, address *js.Object) *js.Object {
```
and then check manually if `address` is `js.Undefined`. Go string may then be extracted with `address.String()`.